app/views/mail/newUser.scala.html

```diff
@@ -4,59 +4,18 @@
 
   <p>Hi @name,</p>
 
+  <p>
+    Welcome to your new webKnossos account. You are almost ready to start exploring and to collaboratively
+    annotate large 3D datasets.
+  </p>
+
```

Comment:
If enableAutoVerify is enabled it now sounds like the account is not ready yet ("You are almost ready ..."). Why did you decide against the if/else distinction depending on enableAutoVerify from the previous version?

Reply:
Because I am confused ;-) Good catch